rdg_datasets/rmat10_hosts=6/README.md

```diff
@@ -0,0 +1,11 @@
+This is a partitioned LSPG format version of rmat10 in format
+DLSPG1.STPG0 and DLSPG2.STPG0.  These are the only partitioned
+graphs in this format in the repository.
+
+# Generation
+
+These graphs were generated via import of the CSV files and then using the
+python remote API to partition the graph.
```

Comment:
It would be nice to have a script or the commands used to allow future users the ability to re-create this from the originals.